app/models/foreman_tasks/task.rb

```diff
@@ -202,6 +203,19 @@ def sub_tasks_counts
       result.symbolize_keys
     end
 
+    def format_input(include_action = false)
```

Comment:
I think we should get rid of `include_action` parameter. It will cause inconsistent output in the `format_task_input` helper, depending on the fact was the task migrated or created by `dynflow_task`.